packages/blazor-workspace/NimbleBlazor/Components/NimbleWaferMap.razor.cs

```diff
@@ -135,6 +138,19 @@ public partial class NimbleWaferMap : ComponentBase
         }
     }
 
+    /// <summary>
+    /// Sets the data in the wafer map.
+    /// </summary>
+    /// <param name="data">The input data, an Apache.Arrow.RecordBatch, which fills the wafer map with content</param>
+    public async Task SetDataAsync(RecordBatch data)
```

Comment:
This doesn't have to be part of this PR, but I think the ideal would be to even get rid of the `Dies` property, and just have a method overload of `SetDataAsync` that takes an `IEnumerable<WaferDie>` as its parameter.

Reply:
Yes, I think that is the direction, but also we have to remove it from the nimble component first

Reply:
The goal of the SetDataAsync method is that it unlike property setting we know when it can be finished. So I was not expecting to see the render-finished event still exist. I'd expect the SetDataAsync method to finish when render has finished.